### PR TITLE
[SRVCOM-1559] Remove outdated sizing requirements

### PR DESCRIPTION
**Pull request by abrennan89**
Jira: https://issues.redhat.com/browse/SRVCOM-1559

Applies for OCP 4.6+

Direct preview link: https://deploy-preview-36964--osdocs.netlify.app/openshift-enterprise/latest/serverless/admin_guide/install-serverless-operator?utm_source=github&utm_campaign=bot_dp

### CODE REVIEW

File: modules/serverless-cluster-sizing-req.adoc [hunk 1]
```diff
@@ -1,27 +0,0 @@
...
-
-In the size requirement provided, an application can scale up to 10 replicas. Lowering the actual CPU request of applications can increase the number of possible replicas.
-
-[NOTE]
-====
-The following limitations apply to all {ServerlessProductName} deployments:
-
-* Maximum number of Knative services: 1000
-* Maximum number of Knative revisions: 1000
-====
```
Comment (cardil) [inline modules/serverless-cluster-sizing-req.adoc:27]: I wonder if we should leave those. This is not minimal requirements, but something we've checked is okay to support.

File: modules/serverless-cluster-sizing-req.adoc [hunk 2]
```diff
@@ -1,27 +0,0 @@
...
-To install and use {ServerlessProductName}, the {product-title} cluster must be sized correctly.
-
-[NOTE]
-====
-The following requirements relate only to the pool of worker machines of the {product-title} cluster. Control plane nodes are not used for general scheduling and are omitted from the requirements.
-====
-
-The minimum requirement for {ServerlessProductName} is a cluster with 10 CPUs and 40GB memory.
-
-The total size requirements to run {ServerlessProductName} are dependent on the applications deployed. By default, each pod requests approximately 400m of CPU, so the minimum requirements are based on this value.
```
Comment (cardil) [inline modules/serverless-cluster-sizing-req.adoc:17]: This paragraph I think is worth keeping as well.

File: serverless/admin_guide/install-serverless-operator.adoc [hunk 3]
```diff
@@ -13,23 +13,6 @@ This guide walks cluster administrators through installing the {ServerlessOperat
...
 
-include::modules/serverless-cluster-sizing-req.adoc[leveloffset=+1]
-
-[id="installing-openshift-serverless-advanced-reqs"]
-== Additional requirements for advanced use cases
-
-For more advanced use cases such as logging or metering on {product-title}, you must deploy more resources. Recommended requirements for such use cases are 24 CPUs and 96GB of memory.
-
-If you have high availability (HA) enabled on your cluster, this requires between 0.5 - 1.5 cores and between 200MB - 2GB of memory for each replica of the Knative Serving control plane.
-HA is enabled for some Knative Serving components by default. You can disable HA by following the documentation on xref:../../serverless/admin_guide/serverless-ha.adoc#serverless-ha[Configuring high availability replicas on {ServerlessProductName}].
```
Comment (cardil) [inline serverless/admin_guide/install-serverless-operator.adoc:24]: This is valid information for sure relates to current state. I think it should be left.

**Review by cardil — CHANGES_REQUESTED** [review]

**Comment by abrennan89** [comment]
Superseded by https://github.com/openshift/openshift-docs/pull/37121

**Pull request closed without merging**